configs/sample_yamls/serviceops/sample_service_config.yml

```diff
@@ -0,0 +1,172 @@
+#
+########################## types ########################
+#
+types:
```

Comment:
I'm OK with deferring as long as:
- we have a tracking bug to make sure we revisit this 
- we can show that the config is legible with types in place

Given that Beta should achieve API/config stability this must be resolved before that release